e2e/portalicious/tests/CreateNewProject/CreateNewProjectSuccessfully.spec.ts

```diff
@@ -18,6 +18,8 @@ test.beforeEach(async ({ page }) => {
 });
 
 test('[29635] Create project successfully', async ({ page }) => {
+  test.fixme(!!process.env.CI, 'Disable use of third-party API in CI only');
```

Comment:
I'm not sure why we are disabling this test. Assuming there are Reasons™, it would be nice to create task to address those reasons, and to link that task here as a "TODO" comment

Reply:
I agree with the above... we can disable them for time-being of course but good idea is to think as well about long lasting solution eg. Mock service or using dev server of Kobo for that part.

Reply:
The main item to address this in a more future-proof way, is currently AB#30220.